docs/releases/1.10.txt

```diff
@@ -55,6 +55,31 @@ Minor features
 :mod:`django.contrib.auth`
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+* Support for the Argon2_ password hash has been added.  Argon2 is the winner
+  of the recent `Password Hashing Competition`_, which has been organized
+  by the community in similar style as the NIST competitions that selected
+  the hash SHA-3 and cipher AES.
```

Comment:
I find this a bit misleading.  It kind of associates SHA-3/AES with Argon2 although only the method (= competition) stayed same.

It would word it more like “…in similar style as the NIST competitions that chose cryptographic standards in the past.”